composer.json

```diff
@@ -12,14 +12,22 @@
     }
   ],
   "require": {
-    "php": "^5.6|^7.0|^8.0"
+    "php": "^7.4|^8.0"
```

Comment:
IMHO, assuming that the package in question follows semver, increasing the minimum PHP version requirement for the package should signify a new major version (i.e., a future JShrink v2.0.0). Considering that there are currently 10 commits pushed to master since the current latest release (JShrink v1.3.3), at the very least, before merging this PR, I reckon it would be worthwhile creating a new tag/release (e.g., JShrink v1.3.4) and doing whatever other preparations might be needed for that. Or, alternatively, if possible, anything belonging to this PR capable of running under the existing minimum PHP version requirement be kept to this PR, while delegating anything which isn't capable to a separate PR to be merged at a later date.

Reply:
Ok, I can add support for version 7.1+.

Older versions do not make sense to support due to missing scalar types.

What do you think?

Reply:
The need for scalar types, which previous PHP versions lack the necessary support for, is a good reason to increase the minimum PHP version requirement, but it's still an increase nonetheless. I also don't necessarily have any problem with increasing the minimum PHP version requirement. But personally, I regard such increases as a kind of "backwards-incompatible change" (since PHP versions which were previously compatible, would no longer be compatible), and because I also like semantic versioning, for at least all of my own PHP packages, I always aim to align any such increases to coincide with new major versions, never changing the minimum PHP version requirement at other times. That's why I make the comment here about that (though, I also note that I am not an existing contributor to this package and have no authority here).

Before changing anything further, I guess, I would ask the maintainers, do they agree or disagree with my thoughts that, increasing minimum PHP version requirement should = new major version release? And, if they agree, should that happen now? Or, should that wait until later (and if so, separate PR to do that, or something else)? Probably, then, we could better determine the best minimum PHP version to set.

Actually, it's not really a problem for me at all, which minimum PHP version is set, since I have access to various different PHP versions at any websites that I manage, multiple versions installed at dev, etc, and I'm not really restricted in that regard. But, it's always good to consider that such problems might potentially exist for others. (In particular, the reason I discovered about JShrink in the first place, is due to a different project which I casually contribute to, which includes JShrink as a dependency, and that particular project itself is locked to PHP 5.6 at the moment).